.github/workflows/build.yml

```diff
@@ -4,6 +4,8 @@ on:
     branches:
       - develop
       - master
+      - v9
```

Comment:
hmm why is this showing up here like this, are these changes not already on the v9 branch? is this just a GH UI problem?

Reply:
yeah super weird, idk 🤔 the changes are definitely on v9